src/stars.css

```diff
@@ -0,0 +1,72 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 :nth-child(1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 :nth-child(2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 :nth-child(3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 :nth-child(1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 :nth-child(2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 :nth-child(3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 :nth-child(4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 :nth-child(1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 :nth-child(2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 :nth-child(3) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 :nth-child(4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 :nth-child(5) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
Looks too complicated, use this documentation to simplify it
https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child
![image](https://github.com/mate-academy/layout_stars/assets/80700247/4dda2c43-3433-4590-b565-aab65f79eb95)